### PR TITLE
Radio Player Component Scaffolding

### PR DESCRIPTION
**Pull request by jbuckner**
**Description**

> Add the radio player component npm scaffolding

**Technical**

> This just creates the npm scaffolding for the radio player component

**Testing**

> n/a

**Evidence**

> n/a

### CODE REVIEW

File: packages/radio-player/package.json [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  "name": "@internetarchive/radio-player",
+  "version": "0.0.0",
+  "description": "A Radio Player LitElement.",
+  "license": "AGPL-3.0-only",
+  "main": "index.js",
+  "module": "index.js",
+  "types": "index.d.ts",
+  "files": [
+    "lib/**/*",
+    "!lib/demo",
```
Comment (iisa) [inline packages/radio-player/package.json]: question: is the `!` intentional?

File: packages/radio-player/package.json [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    "index.d.ts",
+    "index.js"
+  ],
+  "scripts": {
+    "prepare": "npm run build",
```
Comment (iisa) [inline packages/radio-player/package.json]: Suggestion: Use `yarn` to stay in parity with previous iaux components. We also decided yarn was our main package manager, perhaps we need to revisit

File: packages/radio-player/package.json (hunk 1 above)
Comment (jbuckner) [reply packages/radio-player/package.json]: Yeah. This is the list of files that we want to package into the npm module when we upload it to npmjs.io and we want to ignore the demo directory.

File: packages/radio-player/package.json (hunk 2 above)
Comment (jbuckner) [reply packages/radio-player/package.json]: Done

**Pull request merged**